File: com.systemincloud.examples.tasks.console/src/test/java/tasks/data/DataRaw.java
package tasks.data;

import com.systemincloud.modeler.tasks.javatask.api.JavaTask;
import com.systemincloud.modeler.tasks.javatask.api.OutputPort;
import com.systemincloud.modeler.tasks.javatask.api.annotations.JavaTaskInfo;
import com.systemincloud.modeler.tasks.javatask.api.annotations.OutputPortInfo;
import com.systemincloud.modeler.tasks.javatask.api.data.Raw;

@JavaTaskInfo(generator = true)
public class DataRaw extends JavaTask {

	@OutputPortInfo(name = "Out", dataType = Raw.class)
	public OutputPort out;
	
	@Override
	public void generate() {
		byte[] bytes = "String".getBytes();
		out.putData(new Raw(bytes));
	}
}

File: com.systemincloud.examples.tasks.pythontask/src/test/py/tasks/data/DataRaw.py
from sicpythontask.PythonTaskInfo import PythonTaskInfo
from sicpythontask.PythonTask import PythonTask
from sicpythontask.InputPort import InputPort
from sicpythontask.OutputPort import OutputPort
from sicpythontask.data.Raw import Raw

@PythonTaskInfo
class DataRaw(PythonTask):

    def __init_ports__(self):
        self.in_ = InputPort(name="in", data_type=Raw)
        self.out = OutputPort(name="out", data_type=Raw)

    def execute(self, grp):
        inData = self.in_.get_data(Raw)
        inRaw = inData.values
        self.out.put_data(Raw(inRaw))
